suite-native/module-connect-device/src/screens/ConnectAndUnlockDeviceScreen.tsx

```diff
@@ -0,0 +1,32 @@
+import { ImageBackground } from 'react-native';
+
+import { Screen } from '@suite-native/navigation';
+import { Text, Image, VStack } from '@suite-native/atoms';
+import { useNativeStyles, prepareNativeStyle } from '@trezor/styles';
+
+const contentStyle = prepareNativeStyle(_ => ({
+    flex: 1,
+}));
+
+export const ConnectAndUnlockDeviceScreen = () => {
+    const { applyStyle } = useNativeStyles();
+
+    return (
+        <Screen customHorizontalPadding={0} customVerticalPadding={0} hasBottomInset={false}>
+            <ImageBackground
+                // eslint-disable-next-line global-require
+                source={require('../assets/bg.png')}
+                resizeMode="stretch"
+                style={applyStyle(contentStyle)}
+            >
+                <VStack flex={1} justifyContent="flex-end">
+                    <Text variant="titleMedium" textAlign="center">
+                        Connect & unlock your Trezor
+                    </Text>
+                    {/* eslint-disable-next-line global-require */}
+                    <Image source={require('../assets/connectedTrezor.png')} />
```

Comment:
nit: We should provide at least some `accesibilityLabel`

Reply:
https://github.com/trezor/trezor-suite/pull/9450/commits/a6b50cc45117d8b0cb492ed7ed025863bbf72e78